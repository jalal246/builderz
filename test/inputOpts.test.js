import { resolve } from "path";

import perFile from "./utils";

describe.only.each`
  pkgName                      | desc
  ${"input-opts-via-build"}    | ${"inputs from build script"}
  ${"input-opts-via-builderz"} | ${"inputs form builderz property"}
  ${"input-opts-mixed"}        | ${"inputs in build script & builderz property"}
`("tests $pkgName for testing: $desc", ({ pkgName }) => {
  it("passes", async () => {
    const pathPure = resolve(__dirname, "fixtures", pkgName);
    const distPath = resolve(pathPure, "dist");

    await perFile(pathPure, distPath);
  });
});
