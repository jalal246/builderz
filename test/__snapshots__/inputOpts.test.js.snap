// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests input-opts-mixed for testing: inputs in build script & builderz property passes 1`] = `1`;

exports[`tests input-opts-mixed for testing: inputs in build script & builderz property passes 2`] = `
"'use strict';

var foo = function foo() {
  for (var _len = arguments.length, bar = new Array(_len), _key = 0; _key < _len; _key++) {
    bar[_key] = arguments[_key];
  }

  return Promise.resolve(\\"foo \\".concat(bar));
};

module.exports = foo;
"
`;

exports[`tests input-opts-via-build for testing: inputs from build script passes 1`] = `1`;

exports[`tests input-opts-via-build for testing: inputs from build script passes 2`] = `
"'use strict';

var foo = function foo() {
  for (var _len = arguments.length, bar = new Array(_len), _key = 0; _key < _len; _key++) {
    bar[_key] = arguments[_key];
  }

  return Promise.resolve(\\"foo \\".concat(bar));
};

module.exports = foo;
"
`;

exports[`tests input-opts-via-builderz for testing: inputs form builderz property passes 1`] = `1`;

exports[`tests input-opts-via-builderz for testing: inputs form builderz property passes 2`] = `
"'use strict';

var foo = function foo() {
  for (var _len = arguments.length, bar = new Array(_len), _key = 0; _key < _len; _key++) {
    bar[_key] = arguments[_key];
  }

  return Promise.resolve(\\"foo \\".concat(bar));
};

module.exports = foo;
"
`;
